Makefile

```diff
@@ -359,6 +359,10 @@ endif
 	if oc get sc ${STORAGE_CLASS}; then oc delete sc ${STORAGE_CLASS}; fi
 	bash scripts/delete-pv.sh
 
+.PHONY: crc_env_vars
+crc_env_vars: ## set your CRC ENV variables and PATH for 'oc'
+	eval $(crc oc-env)
```

Comment:
I think this is ok, since `crc_storage` is also kind of expects to be a crc, while its not a must. in case of not using CRC crc_env_vars will fail. @dprince thoughts?

Reply:
As an aside, I think the crc_storage target is misnamed, I use it on single node openshift environments. How about a new `local_storage` target, then a `crc_storage` target which depends on `crc_env_vars` and `local_storage`?